Dockerfile

```diff
@@ -27,14 +27,16 @@ RUN yum install -y \
 # configure java runtime
 ENV JAVA_HOME=/opt/java \
   JAVA_VERSION_MAJOR=8 \
-  JAVA_VERSION_MINOR=102 \
-  JAVA_VERSION_BUILD=14
+  JAVA_VERSION_MINOR=112 \
+  JAVA_VERSION_BUILD=15
 
 # configure nexus runtime
+ENV SONATYPE_DIR=/opt/sonatype
 ENV NEXUS_VERSION=3.1.0-04 \
-  NEXUS_HOME=/opt/sonatype/nexus \
+  NEXUS_HOME=${SONATYPE_DIR}/nexus \
   NEXUS_DATA=/nexus-data \
-  NEXUS_CONTEXT=''
+  NEXUS_CONTEXT='' \
+  SONATYPE_WORK=${SONATYPE_DIR}/sonatype-work
```

Comment:
@kellyrob99 Note SONATYPE_WORK is effectively /opt/sonatype/sonatype-work and is not within NEXUS_HOME which is effectively /opt/sonatype/nexus.